### PR TITLE
Add safety warning

### PR DESCRIPTION
**Pull request by skooda**
- Add warning message into Step1
- Update "How it works" info

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
       <p id="tooltip-single" class="tooltip"><span class="key">ENTER</span>&nbsp;will&nbsp;submit, <span class="key">SHIFT</span>&nbsp;+&nbsp;<span class="key">ENTER</span>&nbsp;will&nbsp;create&nbsp;a&nbsp;newline</p>
       <p id="tooltip-multi" class="tooltip"><span class="key">CTRL</span>&nbsp;+&nbsp;<span class="key">ENTER</span>&nbsp;will&nbsp;submit, <span class="key">ENTER</span>&nbsp;will&nbsp;create&nbsp;a&nbsp;newline</p>
       <a id="btn-submit" class="animated fadeIn">ENCRYPT AND SHARE</a>
+      <p id="info" class="warn">Read "How it works" before use to make sure your secret is safe!</p>
```
Comment (ghost) [inline templates/index.html]: Could the `"How it works"` be linked to the help?

Comment (skooda) [reply templates/index.html]: Good point ;)

**Comment by ghost** [comment]
I'm ok with this version.

**Comment by ghost** [comment]
Ok.

**Pull request merged**